packages/dapp-connector/test/WalletApi/Cip30Wallet.test.ts

```diff
@@ -25,20 +33,50 @@ describe('Wallet', () => {
     expect(wallet.apiVersion).toBe('0.1.0');
     expect(typeof wallet.name).toBe('string');
     expect(wallet.name).toBe(testWallet.properties.walletName);
+    expect(wallet.supportedExtensions).toEqual<WalletApiExtension[]>([{ cip: 95 }]);
     expect(typeof wallet.isEnabled).toBe('function');
     const isEnabled = await wallet.isEnabled();
     expect(typeof isEnabled).toBe('boolean');
     expect(isEnabled).toBe(false);
     expect(typeof wallet.enable).toBe('function');
   });
 
-  test('enable', async () => {
-    expect(await wallet.isEnabled()).toBe(false);
-    const api = await wallet.enable();
-    expect(typeof api).toBe('object');
-    const methods = new Set(Object.keys(api));
-    expect(methods).toEqual(new Set(WalletApiMethodNames));
-    expect(await wallet.isEnabled()).toBe(true);
+  describe('enable', () => {
```

Comment:
Users might `enable()`, then `enable({extensions: [{cip: 95}]})`.
I think a test to see if the second call to enable, with the cip95 extension, will return the cip95 methods would be good to have.
```diff
+ await wallet.enable();
+ const api = await wallet.enable({ extensions: [{ cip: 95 }] });
```

Reply:
Done here with the rest of the requested changes: https://github.com/input-output-hk/cardano-js-sdk/pull/887/commits/d17eb7ba288aecd3015a61d1eb9824c001de94d9